src/reuse/modules/common/date.ts

```diff
@@ -23,7 +23,7 @@ export class DateModule {
    * @returns {String} The date in the given format.
    * @example const date = await common.date.getToday("mm/dd/yyyy");
    */
-  getToday(format: DateFormatsType = DateFormats.OBJECT): Date | string {
+  getToday(format: DateFormatsType = process.env.USER_SETTINGS_DATE_FORMAT as DateFormats.OBJECT): Date | string {
```

Comment:
Is it possible that process.env.USER_SETTINGS_DATE_FORMAT is not defined here, for instance if no login happens? If so, I would add a fallback default option, here and in the other places in this module
```suggestion
  getToday(format: DateFormatsType = process.env.USER_SETTINGS_DATE_FORMAT ?  process.env.USER_SETTINGS_DATE_FORMAT as DateFormats.OBJECT : DateFormats.OBJECT): Date | string {
```

Reply:
i think that should already be the case, but would be more clear with the ternary expression